packages/@jsii/runtime/jest.config.ts

```diff
@@ -0,0 +1,17 @@
+import type { Config } from '@jest/types';
+import base from '../../../jest.config';
+
+const config: Config.InitialOptions = {
+  ...base,
+
+  coverageThreshold: {
+    ...base.coverageThreshold,
```

Comment:
NIT: We could use something like [deep-merge](https://www.npmjs.com/package/deepmerge) or write our own little recursive object merge function. I feel like we probably already have one somewhere.